Dcase task1 Baseline

siddhu001 opened the pull request.

siddhu001 commented:
@sw005320 @ftshijt I made this PR to add a baseline and data preparation code for Dcase task1. I believe this PR is ready to be merged. Please let me know any necessary changes that need to be made.

ftshijt commented on egs2/dcase_task1/asr1/local/data.sh:
```diff
@@ -0,0 +1,57 @@
+. ./path.sh
+. ./cmd.sh
+
+DCASE="/ocean/projects/cis210027p/siddhana/TAU-urban-acoustic-scenes-2022-mobile-development/"
```
Could you add this entry to db.sh?

ftshijt commented on egs2/dcase_task1/asr1/local/data.sh:
```diff
@@ -0,0 +1,57 @@
+fi
+
+if [ -z "${DCASE}" ]; then
+    log "Fill the value of 'FSC' of db.sh"
```
Is it really FSC?

siddhu001 replied:
No this was typo, it should be DCASE. Good catch!

ftshijt commented on egs2/dcase_task1/asr1/run.sh:
```diff
@@ -0,0 +1,33 @@
+./asr.sh \
+    --lang en \
+    --ngpu 1 \
+    --stage 12\
+    --stop_stage 13\
```
Beter to remove

ftshijt reviewed (commented):
Please also modify the entry in egs2/README.md

Btw, does Dcase would have different challenges data every year? If so, you may follow the iwslt convention by adding a year number to the recipe

siddhu001 commented:
Thanks a lot @ftshijt for your feedback. Yes, I believe it makes sense to add year number to the recipe. I have addressed all your comments and incorporated your suggestions.

sw005320 reviewed (commented):
LGTM.
My only request is to change the recipe name from `dcase_task1_2022` to `dcase22_task1`

sw005320 commented:
Thanks a lot, @siddhu001!
